exercises/resistor-colors/examples/success-standard/src/ResistorColors.hs

```diff
@@ -0,0 +1,34 @@
+module ResistorColors (value) where
+
+import Data.Char (toUpper)
+
+data Color =
+    Black
+  | Brown
+  | Red
+  | Orange
+  | Yellow
+  | Green
+  | Blue
+  | Violet
+  | Grey
+  | White
+  deriving (Eq, Show, Read)
+
+convert :: Color -> Int
+convert Black = 0
+convert Brown = 1
+convert Red = 2
+convert Orange = 3
+convert Yellow = 4
+convert Green = 5
+convert Blue = 6
+convert Violet = 7
+convert Grey = 8
+convert White = 9
+
+value :: [String] -> Int
```

Comment:
Since we're converting `["black", "brown"]`, I think it would make sense to

1. Have error handling.
2. Return an `Integer`.

Since it makes sense to have error handling, it also makes sense to ask if we should go with `Maybe` or `Either`.

Then `readMaybe` or `readEither` from [`Text.Read`](http://hackage.haskell.org/package/base/docs/Text-Read.html) can come into use.

Reply:
From the tests in specification, I assumed the input will always be valid. Let me know if I should make this change or wait for more discussion (in the problem spec as well).